ftplugin/python/cellmode.vim

```diff
@@ -95,8 +95,8 @@ function! DefaultVars()
   if !exists("b:cellmode_tmux_sessionname") ||
    \ !exists("b:cellmode_tmux_windowname") ||
    \ !exists("b:cellmode_tmux_panenumber")
-    let b:cellmode_tmux_sessionname = GetVar('cellmode_tmux_sessionname', 'ipython')
-    let b:cellmode_tmux_windowname = GetVar('cellmode_tmux_windowname', 'ipython')
+    let b:cellmode_tmux_sessionname = GetVar('cellmode_tmux_sessionname', '')
+    let b:cellmode_tmux_windowname = GetVar('cellmode_tmux_windowname', '')
     let b:cellmode_tmux_panenumber = GetVar('cellmode_tmux_panenumber', '0')
   end
```

Comment:
To avoid breaking the current behavior, I would prefer if we keep the defaults to $ipython:ipython.0 and you just `let b:cellmode_tmux_sessionname=''` in your .vimrc if you want to run in the current tmux session. 

So basically revert line 98 and 99.